rest_device.go

```diff
@@ -421,7 +423,7 @@ func (c RestClient) replaceDeviceACLTemplate(uuid string, wanAclTemplateUuid *st
 		MgmtAclTemplateUUID: mgmtAclTemplateUuid,
 	}
 	req := c.R().SetBody(reqBody)
-	if err := c.Execute(req, http.MethodPost, path); err != nil {
+	if err := c.Execute(req, http.MethodPut, path); err != nil {
```

Comment:
The API catalog defines that endpoint with a PATCH, is MethodPut correct here? https://developer.equinix.com/catalog/network-edgev1#operation/patchDeviceTemplatesbyUuid

Reply:
Good catch. I have made the fix in the new commit.